lib/tasks/bundle.js

```diff
@@ -63,11 +63,16 @@ function main({
             title: 'Build the application',
             async task(ctx = {}) {
                 const args = process.argv.slice(2);
-                if (appMode === 'standalone') {
+                if (buildMode === 'standalone') {
                     const output = await bash('npm', ['run', 'build:standalone', '--', ...args]);
                     ctx.outputBuild = output;
                     return true;
                 }
+                if (buildMode === 'standalone_with_prefix_path') {
```

Comment:
Can you use this kind of syntax ? _standaloneWithPrefix_ etc. ?
We don't use this syntax inside projects.

Reply:
this sort of aligns with npm tasks, and there we have stuff like `check-types`. I don't care, though. Whichever you think is best.

Reply:
check-types not check_types :grin: